Composer/packages/server/src/locales/en-US.json

```diff
@@ -1310,6 +1355,12 @@
   "manifest_version_1edc004a": {
     "message": "Manifest Version"
   },
+  "maximum_f0e8e5e4": {
+    "message": "Maximum"
+  },
+  "maximum_help_text_7f3e6562": {
+    "message": "Maximum help text"
```

Comment:
This is an odd message.

Reply:
These are placeholder message until UX comes up with one.